### ISSUE TITLE
Tests don't run on Windows when running scripts/test.bat

### ISSUE THREAD

**Issue by MattIrv** [Microsoft/azuredatastudio#3812]
It just prints out the following messages:

```
PS C:\repos\sqlopsstudio> .\scripts\test.bat
[getWordRangeAtPosition]: ignoring custom regexp '.*' because it matches the empty string.
Cannot register 'a'. This property is already registered.
DiagnosticCollection with name 'foo' does already exist.
[91m[main 4:57:07 PM][0m Connection Failed
[93m[main 4:57:07 PM][0m unable to open query editor TypeError: Cannot read property 'newSqlEditor' of undefined
Account update handler encountered error: Called with a new account!
(node:13024) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 31): AssertionError [ERR_ASSERTION]: 0 == 1
(node:13024) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:13024) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 32): AssertionError [ERR_ASSERTION]: 0 == 1
(node:13024) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 33): TypeError: Cannot read property 'container' of undefined
WARNING: Promise with no error callback:225
{ exception: null,
  error: Error: no adapter found
    at ExtHostLanguageFeatures._withAdapter (C:\repos\sqlopsstudio\out\vs\workbench\api\node\extHostLanguageFeatures.js:775:56)
    at ExtHostLanguageFeatures.$releaseWorkspaceSymbols (C:\repos\sqlopsstudio\out\vs\workbench\api\node\extHostLanguageFeatures.js:912:18)
    at C:\repos\sqlopsstudio\out\vs\workbench\test\electron-browser\api\testRPCProtocol.js:97:49
    at Object.notifySuccess [as _notify] (C:\repos\sqlopsstudio\out\vs\base\common\winjs.base.js:1191:59)
    at Object.enter (C:\repos\sqlopsstudio\out\vs\base\common\winjs.base.js:867:30)
    at Promise_ctor._run (C:\repos\sqlopsstudio\out\vs\base\common\winjs.base.js:1089:29)
    at Promise_ctor._completed (C:\repos\sqlopsstudio\out\vs\base\common\winjs.base.js:1057:18)
    at ontimeout (timers.js:475:11)
    at tryOnTimeout (timers.js:310:5)
    at Timer.listOnTimeout (timers.js:270:5),
  promise:
   ErrorPromise_ctor {
     _value: Error: no adapter found
    at ExtHostLanguageFeatures._withAdapter (C:\repos\sqlopsstudio\out\vs\workbench\api\node\extHostLanguageFeatures.js:775:56)
    at ExtHostLanguageFeatures.$releaseWorkspaceSymbols (C:\repos\sqlopsstudio\out\vs\workbench\api\node\extHostLanguageFeatures.js:912:18)
    at C:\repos\sqlopsstudio\out\vs\workbench\test\electron-browser\api\testRPCProtocol.js:97:49
    at Object.notifySuccess [as _notify] (C:\repos\sqlopsstudio\out\vs\base\common\winjs.base.js:1191:59)
    at Object.enter (C:\repos\sqlopsstudio\out\vs\base\common\winjs.base.js:867:30)
    at Promise_ctor._run (C:\repos\sqlopsstudio\out\vs\base\common\winjs.base.js:1089:29)
    at Promise_ctor._completed (C:\repos\sqlopsstudio\out\vs\base\common\winjs.base.js:1057:18)
    at ontimeout (timers.js:475:11)
    at tryOnTimeout (timers.js:310:5)
    at Timer.listOnTimeout (timers.js:270:5),
     _isException: false,
     _errorId: 225 },
  handler: undefined,
  id: 225,
  parent: undefined }
[main 4:57:11 PM] Backup: Could not save workspaces.json: Error: ENOENT: no such file or directory, mkdir 'C:\Users\mairvine\AppData\Local\Temp\vsctests\backupservice\d4546ec6-c2cb-4d22-8fa6-b0d0a34497bf\Backups'
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
Provider MSSQL attempted to register but has no metadata
```

my original thought was that this had something to do with the integration test changes but I went back to the commit before those and it's the same

**Comment by kburtram** [comment]
@MattIrv when I run `scripts\test.bat` I see the same output as you report, but I get a `test-results.xml` file that shows all the tests have run and passed.  The CI builds run this script and are reporting 100% pass rate.  The debug "Unit Tests" action is also running the tests (though this doesn't use the script).

I seem to recall at one point the output was a bunch of '......' followed by a pass rate, which was better than what it's currently doing, though the current behavior doesn't seem to be blocking the tests in the lab or debug scenarios.

**Comment by MattIrv** [comment]
I'll fix this while I'm fixing the bugs from #4118. I should just have to change the output format for the tests

**Comment by MattIrv** [comment]
Fixed in the VS Code 1.30.1 merge
